docs/src/writing-tests-csharp.md

```diff
@@ -207,7 +215,9 @@ public class Tests : PageTest
 <TabItem value="mstest">
 
 ```csharp
+using System.Threading.Tasks;
 using Microsoft.Playwright.MSTest;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
```

Comment:
which .NET version are you using? I think from net6.0 `Microsoft.VisualStudio.TestTools.UnitTesting` should be already required via the usings.cs.

Reply:
> which .NET version are you using? I think from net6.0 `Microsoft.VisualStudio.TestTools.UnitTesting` should be already required via the usings.cs.

The output of `dotnet --info`

```
dotnet --info
.NET SDK (reflecting any global.json):
 Version:   6.0.112
 Commit:    d311a58ee1

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.22621
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\6.0.112\
```

Reply:
`PlaywrightTestsMsTest.csproj` in my project:
![image](https://user-images.githubusercontent.com/30647904/208409605-e435fc27-97c6-4859-a834-489798785e57.png)